packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -441,7 +513,7 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
   }
 
   void _handleDismissUpdateValueChanged() {
-    if (widget.onUpdate != null) {
+    if (widget.onUpdate case final DismissUpdateCallback onUpdate) {
```

Comment:
Is this change necessary?

Reply:
Switching back to a simple null check.
I'm also some kind of an "if-case" evangelist myself 😌

Reply:
I did this to reflect what I did with `shouldTriggerDismiss`

@Piinks @nate-thegrate should I also change this part to match with the current preferences on pattern matching usage within the framework ?
https://github.com/ndelanou/flutter/blob/005d193c8d79e1f069fc2056eee8a33daac40e81/packages/flutter/lib/src/widgets/dismissible.dart#L583

Reply:
Yeah, should probably change that one too.